specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.4-preview.1/settings.json

```diff
@@ -0,0 +1,208 @@
+            "200": {
+              "description": "An account setting containing the name of the setting and the value it is set to.",
+              "schema": {
+                "$ref": "#/definitions/Setting"
+              }
+            },
+            "default": {
+              "description": "Key Vault error response describing why the operation failed.",
+              "schema": {
+                "$ref": "common.json#/definitions/KeyVaultError"
+              }
+            }
+          },
+          "x-ms-pageable": {
+            "nextLinkName": "nextLink"
+          },
+          "x-ms-examples": {
+            "GetKeys": {
+              "$ref": "./examples/UpdateSetting-example.json"
+            }
+          }
+        }
+      },
+      "/settings": {
+        "get": {
+          "tags": [
+            "Settings"
+          ],
+          "operationId": "GetSettings",
+          "summary": "List account settings.",
+          "description": "Retrieves a list of all the available account settings that can be configured.",
+          "parameters": [
+            {
+              "$ref": "#/parameters/ApiVersionParameter"
+            }
+          ],
+          "responses": {
+            "200": {
+              "description": "A response message containing a list of account settings with their associated value.",
+              "schema": {
+                "$ref": "#/definitions/SettingsListResult"
+              }
+            },
+            "default": {
+              "description": "Key Vault error response describing why the operation failed.",
+              "schema": {
+                "$ref": "common.json#/definitions/KeyVaultError"
+              }
+            }
+          },
+          "x-ms-pageable": {
+            "nextLinkName": "nextLink"
+          },
+          "x-ms-examples": {
+            "GetKeys": {
+              "$ref": "./examples/ListSettings-example.json"
+            }
+          }
+        }
+      }
+    },
+    "definitions": {
+      "UpdateSettingsRequest": {
+        "properties": {
+          "value": {
+            "type": "string",
+            "description": "The account setting value. True if setting is enabled, false if disabled"
+          }
+        },
+        "description": "The update settings request object.",
+        "required": [
+          "value"
+        ]
+      },
+      "Setting": {
+        "properties": {
+          "name": {
+            "type": "string",
+            "description": "The account setting to be updated"
+          },
+          "value": {
+            "type": "string",
+            "description": "The account setting value. True if setting is enabled, false if disabled"
+          }
+        },
+        "required": [
+          "name",
+          "value"
+        ],
+        "description": "The update settings response object containing the setting name and its value."
+      },
+      "SettingsListResult": {
+        "properties": {
```

Comment:
This needs to define a `nextLink` property as well, per your pageable definition. Generally, all list operations should be pageable unless a team is 100% sure that a returned array will never, ever be long enough to be paged. That's not typical. See https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#collections

Reply:
We are 100% it will never be pageable.

Reply:
Then you need to get rid of the x-pageable on line 140.